### PR TITLE
deleted check for empty to do list

### LINKED ISSUE #9007: [GreenCity][My Space] Retrieving all user's To Do List returns 404 Not Found instead of 200 OK
**Environment:** Windows 11, Chrome
**Reproducible:** always

**Preconditions**
When going to My Space on GreenCity as a User, failing to retrieve all user's To Do List.

**Steps to reproduce**
1. Logging as User
2. Go to GreenCity
3. Click on My Space

**Actual result**
While opening the page of My Space, retrieving all user's To Do List returns 404 Not Found.

**Expected result**
While opening the page of My Space, retrieving all user's To Do List returns 200 OK and response with To Do List.

### PR DESCRIPTION
**Pull request by ChernyavckiiPavlo**
**GreenCity**
[**#9007**](https://github.com/ita-social-projects/GreenCity/issues/9007)
Since it is possible that a user will not have any habits and, accordingly, their to-do list will be empty, the check for an empty list, which returned a 404 error, was removed, as such cases are handled on the front end.

## Summary by CodeRabbit

* **Bug Fixes**
  * Viewing in-progress habits/to‑do lists now returns an empty list instead of raising an error when no items are in progress.

* **Tests**
  * Removed a test that expected an error for empty in‑progress results (edge-case expectation dropped).

* **Chores**
  * Removed a legacy error message constant related to in‑progress habit assigns.

### CODE REVIEW

**Review by oleksandr-ilnytskyi — APPROVED** [review]

File: service/src/main/java/greencity/service/HabitAssignServiceImpl.java [hunk 1]
```diff
@@ -632,10 +632,7 @@ public UserToDoAndCustomToDoListsDto getUserToDoAndCustomToDoLists(
     public List<UserToDoAndCustomToDoListsDto> getListOfUserAndCustomToDoListsWithStatusInprogress(
         Long userId, String language) {
         List<HabitAssign> habitAssignList = habitAssignRepo.findAllByUserIdAndStatusIsInProgress(userId);
-        if (habitAssignList.isEmpty()) {
-            throw new NotFoundException(
-                ErrorMessage.HABIT_ASSIGN_NOT_FOUND_WITH_CURRENT_USER_ID_AND_INPROGRESS_STATUS + userId);
```
Comment (RomaMocherniuk) [inline service/src/main/java/greencity/service/HabitAssignServiceImpl.java:637]: Please remove `HABIT_ASSIGN_NOT_FOUND_WITH_CURRENT_USER_ID_AND_INPROGRESS_STATUS` costant as well

**Review by RomaMocherniuk — APPROVED** [review]

**Review by BohdanLys — APPROVED** [review]

**Review by markotymofeiuk — APPROVED** [review]

File: service/src/main/java/greencity/service/HabitAssignServiceImpl.java (hunk 1 above)
Comment (BohdanLys) [reply service/src/main/java/greencity/service/HabitAssignServiceImpl.java:637]: Fixed it
